src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyCatchEmptyComment.java

```diff
@@ -0,0 +1,98 @@
+package com.google.checkstyle.test.chapter4formatting.rule413emptyblocks;
+
+import java.io.IOException;
+
+/** some javadoc. */
+public class InputFormattedEmptyCatchEmptyComment {
+  private void foo() {
+    try {
+      throw new RuntimeException();
+      // violation below 'Empty catch block'
+    } catch (Exception expected) {
+      //
+    }
+  }
+
+  private void foo2() {
+    try {
+      throw new IOException();
+    } catch (IOException | NullPointerException | ArithmeticException ignore) {
+      // ignore
+    }
+  }
+
+  private void foo3() { // comment
+    try {
+      throw new IOException();
+      // violation below 'Empty catch block'
+    } catch (IOException | NullPointerException | ArithmeticException e) {
+      /**/
+    }
+  }
+
+  private void foo4() {
+    try {
+      throw new IOException();
+      // violation below 'Empty catch block'
+    } catch (IOException | NullPointerException | ArithmeticException e) {
+      /* */
+    }
+  }
+
+  private void foo5() {
+    try {
+      throw new IOException();
+    } catch (IOException | NullPointerException | ArithmeticException e) { //
+    } // violation above 'Empty catch block'
+  }
+
+  private void foo6() {
+    try {
+      throw new IOException();
+    } catch (IOException e) { //
+    } // violation above 'Empty catch block'
```

Comment:
please put all `// violation ....` under throw.
It is very clear pattern to read such inputs, I like how you did this in other cases below. Please apply to all new snippets.